packages/terra-action-header/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-action-header",
   "main": "lib/ActionHeader.js",
-  "version": "2.74.3",
+  "version": "2.75.0",
```

Comment:
Should we use a patch version (i.e. 2.74.4) for all these `Fixed broken link in documentation` changes instead of a minor version since there is really no actual code changes in the components? Consumers won't even see this change in terra-ui because the change was only made in the README.

Reply:
+1 to this suggestion

Reply:
It's probably not even worth doing a release for these `Fixed broken link in documentation` changes. 

I think we should just remove the entry from the changelog to avoid doing a release all of these components.